composer.json

```diff
@@ -22,6 +22,7 @@
 		"squizlabs/php_codesniffer": "3.10.3",
 		"wp-coding-standards/wpcs": "~3.1.0",
 		"phpcompatibility/phpcompatibility-wp": "~2.1.3",
+		"phpstan/phpstan": "^2.1.2",
```

Comment:
Depending on when this PR gets merged before or after https://core.trac.wordpress.org/ticket/62622 we may need to drop this down to phpstan v1.x.

I'm leaving it at a higher version for now, so I can continue to use the latest PHPStan version for remediation over in https://github.com/justlevine/wordpress-develop/tree/chore/phpstan/level-0